src/block/count-up/edit.js

```diff
@@ -60,7 +62,21 @@ const Edit = props => {
 		<Fragment>
 
 			<InspectorTabs />
+			<InspectorBlockControls>
```

Comment:
I feel that this control is better suited to be inside the **Style tab** and as the first panel and should be open by default